src/cpp/session/modules/rmarkdown/NotebookAlternateEngines.R

```diff
@@ -116,6 +116,7 @@
    
    # when invoking engines, we don't want to echo user code
    mergedOptions$echo <- FALSE
+   mergedOptions$message <- FALSE
```

Comment:
Curious what this does.

Reply:
The intention here is to avoid the console output produced by `knitr` when running an engine, e.g.

https://github.com/yihui/knitr/blob/d0d1b470e6bd5133361e54ec78c3f3d1b1ce629f/R/engine.R#L213

I should add a comment to make that clear too.